routes/api/project/location/load_balancer.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CloverApi
+  hash_branch(:project_location_prefix, "load-balancer") do |r|
+    r.on String do |lb_name|
+      pss = @project.private_subnets_dataset.where(location: @location).all
+      lb = pss.flat_map { _1.load_balancers_dataset.where { {Sequel[:load_balancer][:name] => lb_name} }.all }.first
+
+      unless lb
+        response.status = 404
+        r.halt
+      end
+
+      load_balancer_endpoint_helper = Routes::Common::LoadBalancerHelper.new(app: self, request: r, user: @current_user, location: @location, resource: lb)
```

Comment:
Do we use location anywhere in `LoadBalancerHelper`?

Reply:
Removed it